mapper-runtime/revapi.json

```diff
@@ -60,8 +60,13 @@
       {
         "code": "java.method.addedToInterface",
         "new": "method java.lang.String com.datastax.oss.driver.api.mapper.MapperContext::getExecutionProfileName()",
-        "justification": "JAVA-2633 Adding execution profile to mapper"
+        "justification": "JAVA-2633: Add execution profile argument to DAO factory method (accept API break -- it's unlikely that MapperContext will be implemented outside of the driver)"
```

Comment:
Can't we use `customState` to propagate this? We would not need to break backward compatibility in such a case

Reply:
Already discussed [on the original PR](https://github.com/datastax/java-driver/pull/1409#discussion_r389558576).

To expand: `MapperContext` can't be implemented by the user in any useful way (they can't inject their implementation in mapper-generated code), so we deemed it acceptable to break the API.